news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Beatmap Spotlights: Winter 2020"
+date: 2020-03-20 08:00:00 +0000
```

Comment:
```suggestion
date: 2020-03-31 08:00:00 +0000
```

Reply:
Please re-date everything to this date. Our target date is always either the last day of the month or the first day of the next.